lib/components/Application.js

```diff
@@ -18,6 +18,7 @@ export default class Application extends Component {
       user: null,
       activeUsers: [],
       searchField: 'hi',
+      charCount: 0,
```

Comment:
I think we can remove this state field with the new design.  Want to update and push back up?